packages/x-components/src/components/layouts/fixed-header-and-asides-layout.vue

```diff
@@ -95,7 +95,7 @@
   @Component({
     components: {
       BaseIdModal,
-      BaseIdScroll
+      Scroll
```

Comment:
Importing a component from a module in a non module component shouldn't be allowed. But I think we can discuss the solution to this problem and move it into another PR. One solution would be to allow dynamic root components in our layouts sections.

Reply:
I think in this case in particularly we're always going to have scroll right? It's for me but thanks for pointing out.

Reply:
The problem is that even if you don't use our layouts you will still have the scroll module loaded.